go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/mre/riffraff
+
+require (
+	github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc // indirect
+	github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf // indirect
+	github.com/bndr/gojenkins v0.2.1-0.20180824070327-ddda3b6176a8
```

Comment:
Did you intend to upgrade `gojenkins` from `21e24a455945cd92fd39da7a3576f2e6e5a2ddae` to  `ddda3b6176a8`?